scripts/ci/selective_tests.sh

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# shellcheck source=scripts/ci/libraries/_script_init.sh
+. ./scripts/ci/libraries/_script_init.sh
+
+declare -a pattern_array
+
+function output_all_basic_variables() {
+    initialization::ga_output python-versions \
+        "$(initialization::parameters_to_json "${CURRENT_PYTHON_MAJOR_MINOR_VERSIONS[@]}")"
+    initialization::ga_output default-python-version "${DEFAULT_PYTHON_MAJOR_MINOR_VERSION}"
+
+    initialization::ga_output kubernetes-versions \
+        "$(initialization::parameters_to_json "${CURRENT_KUBERNETES_VERSIONS[@]}")"
+    initialization::ga_output default-kubernetes-version "${KUBERNETES_VERSION}"
+
+    initialization::ga_output kubernetes-modes \
+        "$(initialization::parameters_to_json "${CURRENT_KUBERNETES_MODES[@]}")"
+    initialization::ga_output default-kubernetes-mode "${KUBERNETES_MODE}"
+
+    initialization::ga_output postgres-versions \
+        "$(initialization::parameters_to_json "${CURRENT_POSTGRES_VERSIONS[@]}")"
+    initialization::ga_output default-postgres-version "${POSTGRES_VERSION}"
+
+    initialization::ga_output mysql-versions \
+        "$(initialization::parameters_to_json "${CURRENT_MYSQL_VERSIONS[@]}")"
+    initialization::ga_output default-mysql-version "${MYSQL_VERSION}"
+
+    initialization::ga_output kind-versions \
+        "$(initialization::parameters_to_json "${CURRENT_KIND_VERSIONS[@]}")"
+    initialization::ga_output default-kind-version "${KIND_VERSION}"
+
+    initialization::ga_output helm-versions \
+        "$(initialization::parameters_to_json "${CURRENT_HELM_VERSIONS[@]}")"
+    initialization::ga_output default-helm-version "${HELM_VERSION}"
+
+    initialization::ga_output postgres-exclude '[{ "python-version": "3.6" }]'
```

Comment:
If we have as many savings as I think we will, we can likely even bring back those python variants.